examples/example_batch_execution.py

```diff
@@ -40,15 +41,16 @@ def create_preprocessing_pipeline(power_scaler):
 def create_test_pipeline(regressor_svr):
     # Create test pipeline which works on a batch size of one hour.
     pipeline = Pipeline("../results/test_pipeline", batch=pd.Timedelta("1h"))
-
+    periodic_condition = PeriodicCondition(21)
     # Add the svr regressor to the pipeline. This regressor should be called if it is not daytime
     regressor_svr_power_statistics = regressor_svr(historical_input=pipeline["historical_input"],
                                                    target=pipeline["load_power_statistics"],
                                                    computation_mode=ComputationMode.Refit,
                                                    callbacks=[LinePlotCallback('SVR')],
                                                    lag=pd.Timedelta(hours=24),
-                                                   train_if=lambda x, _: len(x["historical_input"].indexes["time"]) > 0 and
-                                                                         x["historical_input"].indexes["time"][0].hour == 0)
+                                                   refit_conditions=[periodic_condition,
+                                                       lambda x, _: len(x["historical_input"].indexes["time"]) > 0 and
+                                                                         x["historical_input"].indexes["time"][0].hour == 0])
```

Comment:
reformat lines
PEP 8: E128 continuation line under-indented for visual indent
PEP 8: E501 line too long (125 > 120 characters)